FRONTEND/src/components/About/ValuesSection.jsx

```diff
@@ -31,7 +33,9 @@ const ValuesSection = () => {
           {/* Benefit 3 */}
           <div className="p-6 bg-white rounded-lg shadow-md hover:shadow-lg transition-shadow duration-300">
             <FaRegChartBar className="text-4xl text-blue-600 mx-auto mb-4" />
-            <h3 className="text-2xl font-semibold mb-3">Make out-of-Home easy.</h3>
+            <h3 className="text-2xl font-semibold mb-3">
+              Make out-of-Home easy.
```

Comment:
Why are there "." full stops in some values? Make it consistent by removing them

Reply:
Done

Reply:
Done